features/nationaliteit.feature

```diff
@@ -2,319 +2,278 @@
+      | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
+      | 0100                  | 301                   | 20200713                        |
+      En de 'nationaliteit' is gecorrigeerd naar de volgende gegevens
+      | onjuist (84.10) | datum ingang geldigheid (85.10) |
+      | O               | 20200727                        |
+      En de persoon heeft nog een 'nationaliteit' met de volgende gegevens
+      | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
+      | 0263                  | 301                   | 20100801                        |
       Als personen wordt gezocht met de volgende parameters
-      | naam                | waarde                          |
-      | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 999994748                       |
-      | fields              | nationaliteiten                 |
+      | naam                | waarde                              |
+      | type                | RaadpleegMetBurgerservicenummer     |
+      | burgerservicenummer | 000000085                           |
+      | fields              | nationaliteiten                     |
       Dan heeft de response een persoon met een 'nationaliteit' met de volgende gegevens
-      | naam | waarde                     |
-      | type | VastgesteldNietNederlander |
+      | naam                              | waarde                                               |
+      | type                              | Nationaliteit                                        |
+      | nationaliteit.code                | 0263                                                 |
+      | nationaliteit.omschrijving        | Surinaamse                                           |
+      | redenOpname.code                  | 301                                                  |
+      | redenOpname.omschrijving          | Vaststelling bezit vreemde nationaliteit             |
+      | datumIngangGeldigheid.type        | Datum                                                |
+      | datumIngangGeldigheid.datum       | 2010-08-01                                           |
+      | datumIngangGeldigheid.langFormaat | 1 augustus 2010                                      |
+
 
-    @proxy
-    Scenario: de persoon is vastgesteld niet-Nederlander, nationaliteit heeft code 0500
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 555550002 |
-      En de persoon heeft een 'nationaliteit' met de volgende gegevens
-      | naam               | waarde |
-      | nationaliteit.code | 0500   |
+    Scenario: geactualiseerde ingangsdatum geldigheid
+      Gegeven de persoon met burgerservicenummer '000000097' heeft een 'nationaliteit' met de volgende gegevens
+      | nationaliteit (05.10) | reden opnemen (63.10) | onjuist (84.10) | datum ingang geldigheid (85.10) |
```

Comment:
Hier verwacht ik het volgende:
      Gegeven de persoon met burgerservicenummer '000000097' heeft een 'nationaliteit' met de volgende gegevens
      | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
      | 0100                  | 311                   | 20200713                        |
      En de 'nationaliteit' is gecorrigeerd naar de volgende gegevens
      | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
      | 0100                  | 311                   | 20200727                        |
Maar ik weet niet of dit is wat hier functioneel wordt beschreven

Reply:
Aangepast